src/components/container/ExternalAccountConnectionAlert/ExternalAccountConnectionAlert.tsx

```diff
@@ -0,0 +1,65 @@
+import MyDataHelps, { ExternalAccount } from "@careevolution/mydatahelps-js";
+import React, { useState } from "react";
+import language from "../../../helpers/language";
+import { Action } from "../../presentational";
+import { FontAwesomeSvgIcon } from "react-fontawesome-svg-icon";
+import { faTriangleExclamation } from "@fortawesome/free-solid-svg-icons";
+import { useInitializeView } from "../../../helpers/Initialization";
+
+export interface ExternalAccountConnectionAlertProps {
+    previewState?: "externalAccountWithIssue" | "externalAccountNoIssue"
+    onClick(): void
+    text?: string
+    externalAccountCategories?: string[];
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+export default function (props: ExternalAccountConnectionAlertProps) {
+    let [externalAccounts, setExternalAccounts] = useState<ExternalAccount[]>([]);
+
+    useInitializeView(() => {
+        if (props.previewState === 'externalAccountNoIssue') {
+            setExternalAccounts([{
+                id: 1,
+                provider: {
+                    id: 1,
+                    name: "Provider",
+                    category: "Provider",
+                    logoUrl: "https://google.com"
```

Comment:
Placeholder that needs to be replaced? Also below.

Reply:
Fixed